requirements-dev.txt

```diff
@@ -1,3 +1,11 @@
 flake8
 pytest
 build
+tox
+isort
+mypy
+black==22.6.0
+click==8.0.3
+bandit
+pytest
+pytest-cov
```

Comment:
Are these intended to be here? Tox will install the specified `deps` to separate virtual envs, they don't need to be duplicated here

(alternatively, they can be kept here and don't need to be duplicated in tox.ini)

Reply:
This is just there so the developer can test outside of tox, using venv (and apply the formatting commands).

Reply:
but tox is the easy way developers to test it

Reply:
Maybe we can have the best of both worlds by maintaining the tox deps _only_ in this file? And remove all the duplicated `deps` sections from tox.ini?